CMake/kwiver-flags-gnu.cmake

```diff
@@ -116,3 +116,9 @@ if (KWIVER_CPP_COVERAGE   AND   CMAKE_BUILD_TYPE EQUAL "DEBUG")
   # It seems as though the flag isn't detected alone.
   kwiver_check_compiler_flag(-fprofile-arcs)
 endif ()
+
+# GCC Flag used for stripping binaries of any unused symbols
+# Used for reducing the size of kwiver wheel since pypi has 60Mb constraint on wheel size
```

Comment:
This comment is very helpful. So is the one for zlib. It helps me know what its doing and why I want to be doing it.

Reply:
Yes, I agree that it does help with the explanation. Originally I was using commit messages or more detailed descriptions to document the caveats. But I did drop the ball on these two changes.

Reply:
While descriptive git messages are nice, my opinion is that in-code comments are better because they are more tightly coupled with the code. The git history can be unintentionally overwritten and its not always intuitive that an explanation for a cryptic line of code might be in the git history. However, in-code comments are much harder to miss and unintentionally remove. 

That being said, I'm certainly no paragon of [good git commit messages](https://github.com/Erotemic/ibeis/commits/master).